wobbly-glib/model.cpp

```diff
@@ -0,0 +1,447 @@
+/*
+ * wobbly-glib/wobbly.cpp
+ *
+ * GObject Interface for "wobbly" textures
+ *
+ * See LICENCE.md for Copyright information
+ */
+
+#include <wobbly-glib/anchor.h>
+#include <wobbly-glib/model.h>
+#include <wobbly-glib/vector.h>
+
+#include <wobbly/wobbly.h>
+
+#include "wobbly-anchor-private.h"
+
+struct _WobblyModel
+{
+  GObject parent_instance;
+};
+
+typedef struct _WobblyModelPrivate
+{
+  wobbly::Model::Settings  settings;
+  wobbly::Model           *model;
+
+  /* We need to keep track of these during
+   * construction while model isn't set */
+  WobblyVector             prop_position;
+  WobblyVector             prop_size;
```

Comment:
If these are only used for that purpose, I think you could as well use `constructor()` instead of `constructed()`, since you will get the array of construct property in there. But up to you if that does not actually simplify things.

Reply:
That would be handy - means we don't have don't have to store all this stuff in memory